app/services/mcp_presets.py

```
@@ -0,0 +1,153 @@
+"""Server-side MCP presets for trusted local mcpproxy groups."""
+
+import os
+from typing import Any, Mapping
```

Ruff (0.15.20)

[warning] Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)